analyzer/runtime/extract.go

```diff
@@ -364,15 +364,40 @@ func ExtractRound(blockHeader nodeapi.RuntimeBlockHeader, txrs []nodeapi.Runtime
 					blockTransactionData.Body = body
 					amount = body.Amount.Amount
 					if body.To != nil {
-						// Beware, this is the address of an account in the consensus
-						// layer, not an account in the runtime that generated this event.
+						// This is the address of an account in the consensus
+						// layer, not an account in the runtime that generated this tx.
 						// We do not register it as a preimage.
 						if to, err = uncategorized.StringifySdkAddress(body.To); err != nil {
 							return fmt.Errorf("to: %w", err)
 						}
 					}
 					return nil
 				},
+				ConsensusAccountsDelegate: func(body *consensusaccounts.Delegate) error {
+					blockTransactionData.Body = body
+					amount = body.Amount.Amount
+					// This is the address of an account in the consensus
+					// layer, not an account in the runtime that generated this tx.
+					// We do not register it as a preimage.
+					if to, err = uncategorized.StringifySdkAddress(&body.To); err != nil {
+						return fmt.Errorf("to: %w", err)
+					}
+					return nil
```

Comment:
This is analogous to (and copy pasted from) our existing handling of `consensusaccounts.Withdraw` calls

Reply:
:+1: